### ISSUE TITLE
Support something else than React?

### ISSUE THREAD

**Issue by ergo** [zzarcon/react-scroll-shadow#2]
Hi, is there a chance that common functionality can be split out so this module could use used with other frameworks too?

**Comment by zzarcon** [comment]
Hi @ergo that sounds like a good idea. 

If we do so in a way that we can reuse it easily, we can just make react-scroll-shadow to depend on that other lib.

We should probably came up first with an api for that library, maybe something like:

```javascript
import ScrollShadow from 'scroll-shadow'

const shadows = new ScrollShadow(document.body, {/*options*/})

shadows.listen();
shadows.remove();
```

Thoughts?

**Comment by ergo** [comment]
Yeah, sounds good, I was thinking about wrapping this into web component.
But that means that in that example `document.body` would be just a node somewhere, It would not have access to document.body I think.
